packages/product-feed-persoo/src/Resources/views/feed.xml.twig

```diff
@@ -0,0 +1,45 @@
+{% block begin %}
+<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0" xmlns:g="http://base.google.com/ns/2.0">
+    <channel>
+{% endblock %}
+
+{% block item %}
+    <product>
+        <title><![CDATA[{{ item.title }}]]></title>
+        <link>{{ item.link }}</link>
+        <description><![CDATA[{{ item.description|striptags|replace({"\n": ' '})|trim }}]]></description>
+        <id>{{ item.id }}</id>
+        {% if item.itemGroupId is not null %}
+            <item_group_id>{{ item.itemGroupId }}</item_group_id>
+        {% endif %}
+        {% for hierarchyIds in item.hierarchyIdsIndexedByCategoryId %}
+            <hierarchy>{{ hierarchyIds }}</hierarchy>
+        {% endfor %}
+        {% for hierarchyNames in item.hierarchyNamesIndexedByCategoryId %}
+            <hierarchy_text>{{ hierarchyNames }}</hierarchy_text>
+        {% endfor %}
+        {% if item.imageLink is not null -%}
+            <image_link>{{ item.imageLink }}</image_link>
+        {% endif -%}
+        <availability>{{ item.availability }}</availability>
+        <price>{{ item.price.priceWithVat|moneyFormat(2) }} {{ item.currency.code }}</price>
+        {%  if item.brand is not null -%}
+            <brand>{{ item.brand }}</brand>
+        {% endif -%}
+        {% for flagName in item.flagNames %}
+            <tag_{{ flagName }}>true</tag_{{ flagName }}>
```

Comment:
...ah, this is probably the reason for the transformation, right?

Reply:
Yes